grammars/d.cson

```diff
@@ -926,23 +926,23 @@
           ]
         },
         {
-          'match': 'out',
+          'match': 'out\\b',
           'name': 'keyword.control.out.d'
         },
       ]
     },
     'body-statement': {
       'patterns': [
         {
-          'match': 'body',
+          'match': 'body\\b',
           'name': 'keyword.control.body.d'
         },
       ]
     },
     'constructor': {
       'patterns': [
         {
-          'match': 'this',
+          'match': 'this\\b',
```

Comment:
`this` can be immediately followed by a parentheses, as in `this(uint width, uint height) {`